heroku-20/setup.sh

```diff
@@ -185,6 +185,7 @@ apt-get install -y --no-install-recommends \
     patch \
     postgresql-client-12 \
     python3 \
+    python-is-python3 \
```

Comment:
```suggestion
    python-is-python3 \
    python3 \
```

(For consistent package sort order with the generated files)